data/json/mapgen/microlab/microlab_special_tiles.json

```
@@ -0,0 +1,176 @@
+        "  c c|c   |22|    |cc   "
+      ],
+      "palettes": [ "microlab" ],
+      "terrain": { "G": "t_card_science" },
+      "items": { "f": { "item": "mutagens", "chance": 60, "repeat": [ 1, 4 ] } },
+      "monster": { "M": { "monster": "mon_mutant_experimental" } },
+      "place_monsters": [ { "monster": "GROUP_LAB", "chance": 2, "x": [ 2, 21 ], "y": [ 2, 21 ], "repeat": [ 1, 5 ] } ]
+    }
+  },
+  {
+    "type": "mapgen",
+    "om_terrain": [ [ "microlab_generic" ] ],
+    "method": "json",
+    "object": {
+      "fill_ter": "t_strconc_floor",
+      "rows": [
+        "     | cc |  | cc |     ",
+        "     2    2  2    | c c ",
+        "   F|||||||  |||||||c c ",
+        "   F|m 5  |22|0c7c0|c c ",
+        "   F|(((  (  |     |    ",
+        "|||||m 5  (  |((2((|  ||",
+        "  66|(((  (       c|    ",
+        "    |m 5  (       6|  c ",
+        "  cc||||  |((((   c|  c ",
+        "    |  2  5   (    |    ",
+        "2((||2||((|  m(    ||2||",
+        "   2      (((((    2    ",
+        "   2               2    ",
+        "2((||              ||2||",
+        "   c|      d6d     |c   ",
+        " c c| d6d   h   d6d|c c ",
+        " c c|  h         h |c c ",
+        "   c|              |F   ",
+        "|||||              |||2|",
+        "c66c|FFFFFF  |ccccc|cc  ",
+        " c c|||||||22|||||||    ",
+        " c cc(ccc |  |FFFF|  cc ",
+        " c        2  ||22||  cc ",
+        "     (    |  |    |     "
+      ],
+      "palettes": [ "microlab" ],
+      "terrain": { "0": "t_vat" },
+      "monster": { "T": { "monster": "mon_mutant_experimental" } },
+      "place_items": [  ],
+      "place_monsters": [ { "monster": "GROUP_LAB", "chance": 2, "x": [ 2, 21 ], "y": [ 2, 21 ], "repeat": [ 1, 5 ] } ]
+    }
+  },
+  {
+    "type": "mapgen",
+    "om_terrain": [ [ "microlab_generic" ] ],
+    "method": "json",
+    "object": {
+      "fill_ter": "t_strconc_floor",
+      "rows": [
+        "     | cc 2  2 cc |     ",
+        "     |    |^^|    |cccc ",
+        "  ||||||||||||||||||||c ",
+        "  |------------------|6 ",
+        "  |-F   cccccc -lll -|r ",
+        "|||-F          2   c-||2",
+        "  |---  cc cc  -  h6-|  ",
+        "  |-F   cc cc  - h c-|c ",
+        "  |-F   cc cc  -cccc-|c ",
+        "  |---         -((((-|  ",
+        "2|G            2   7-|||",
+        "  5  T----22----  @@-|  ",
+        "  5  T-h       5   7-|  ",
+        "||-lll-66      5    -|||",
+        " |-----(((222---------|c",
+        " |-0 0 0 0   0 0 0 0 -| ",
+        " |-0 0 0 0   0 0 0 0 -| ",
+        "c|-                  -|c",
+        "||-0 0 0 0 0 0 0 0 0 -||",
+        "c|-0 0 0 0 0 0 0 0 0 -|F",
+        " |--------------------|F",
+        " |||||||||||||||||||||||",
+        " cccc|c   |^^|    |  cc ",
+        "    c|c   2  2    |     "
+      ],
+      "palettes": [ "microlab" ],
+      "terrain": { "0": "t_vat", "G": "t_card_science" },
+      "monster": { "T": { "monster": "mon_turret" } },
```

```suggestion
      "monster": { "T": { "monster": "mon_turret_rifle" } },
```
9mm turrets are being phased out. See #33436.

---

True I wrote these specifics maps more than a year ago, and forgot to update